packages/botfuel-dialog/src/dialogs/intent-resolution-dialog.js

```diff
@@ -13,32 +13,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const logger = require('logtown')('IntentResolutionDialog');
-const Dialog = require('./dialog');
+const TextDialog = require('./text-dialog');
 
 /**
  * The default resolution dialog when multiple intities are detected from the trainer
  * @extends IntentResolutionDialog
  */
-class IntentResolutionDialog extends Dialog {
-  /**
-   * @constructor
-   * @param {Object} config - the bot config
-   * @param {class} brain - the bot brain
-   */
-  constructor(config, brain) {
-    super(config, brain, { reentrant: false });
-  }
-
-  /** @inheritDoc */
-  async execute(adapter, userMessage, messageEntities) {
-    logger.debug('execute', userMessage, messageEntities);
-    const { entities, intents } = messageEntities;
-    const dialogData = { entities, intents };
-    await this.display(adapter, userMessage, dialogData);
-
-    return this.complete();
-  }
-}
+class IntentResolutionDialog extends TextDialog {}
```

Comment:
Maybe this is in another PR but I think that we can find a better name for this class